dev_scripts/find_images_to_rebuild.py

```diff
@@ -15,12 +16,19 @@
 os.environ["FRESHMAKER_CONFIG_FILE"] = os.path.join(sys.path[0], "config.py")
 os.environ["REQUESTS_CA_BUNDLE"] = "/etc/ssl/certs/ca-bundle.crt"
 
-from freshmaker import db, app
+from freshmaker import db, app, conf
 from freshmaker.errata import Errata, ErrataAdvisory
 from freshmaker.events import (
     ErrataAdvisoryStateChangedEvent, ManualRebuildWithAdvisoryEvent)
 from freshmaker.handlers.koji import RebuildImagesOnRPMAdvisoryChange
 
+parser = argparse.ArgumentParser()
+parser.add_argument('errata_event_id', type=int, help='Errata event ID')
+parser.add_argument('--cassette-path', default=False, dest='cassette',
+                    help='Set a path to a cassette')
+parser.add_argument('--container-images', nargs='+', default=False, dest='container_imgs',
```

Comment:
I don't think this should be plural since you have to specify it multiple times to pass many container images
```suggestion
parser.add_argument('--container-image', nargs='+', default=False, dest='container_imgs',
```